korangar_networking/README.md

```diff
@@ -13,7 +13,7 @@ It connects to the server as a client and uses `Ollama` to generate responses to
 To run this example you need to set `USERNAME`, `PASSWORD`, and `CHARACTER_NAME` in the source code (you might also have to adjust `OLLAMA_ENDPOINT` and `OLLAMA_MODEL` depending on your setup).
 Afterwards you can run it with
 ```bash
-cargo run --example ollama-chat-bot
+cargo run --example ollama-chat-bot --features=example
```

Comment:
Can you maybe explain what the advantage of this feature is over having the `dev-dependencies` as non optional?

I think regardless I would prefer not to require any features when running the examples.

Reply:
My main goal was to reduce build times when running tests. When running all tests inside the workspace and not having this feature, we would need to include the dependencies that are now behind this feature, even though they are only used inside the example and not helpfull in general.

Reply:
While I obviously like reduced build times, for tests I value ease of use over build times. So I would prefer if we removed this feature again and just include the dependencies unconditionally.

Reply:
Fine by me! Should I change it or do you want to change it?

Reply:
I will leave it to you :D

Reply:
Done!